jvm-packages/xgboost4j-gpu/src/test/java/ml/dmlc/xgboost4j/gpu/java/BoosterTest.java

```diff
@@ -83,7 +85,16 @@ public void testBooster() throws XGBoostError {
 
     try (Table tmpTable = Table.readCSV(schema, opts,
 					new File(trainingDataPath))) {
-      CudfColumnBatch batch = new CudfColumnBatch(tmpTable, new int[]{0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11}, new int[]{12});
+      ColumnVector[] df = new ColumnVector[12];
+      for (int i = 0; i < 12; ++i) {
```

Comment:
Is there a functional way to do this?

Reply:
I glanced through the java binding of cuDF, there's a `getColumns` method but it's private.  @wbo4958 might know better than I do.